dataplane/linux/vxlan_mgr.go

```diff
@@ -127,7 +132,10 @@ func newVXLANManager(
 			deviceRouteSourceAddress net.IP, deviceRouteProtocol netlink.RouteProtocol, removeExternalRoutes bool) routeTable {
 			return routetable.New(interfaceRegexes, ipVersion, vxlan, netlinkTimeout,
 				deviceRouteSourceAddress, deviceRouteProtocol, removeExternalRoutes, 0,
-				opRecorder)
+				opRecorder,
+				routetable.WithAddiionalLogFields(log.Fields{"route_table": "vxlan_noencap"}),
+				routetable.WithAdditionalRouteFilters(&netlink.Route{Type: noencTarget.RouteType()}),
```

Comment:
Use `routetable.TargetTypeNoEncap` instead of `noencTarget.RouteType()`.

Reply:
they're not the same types since Type will need something like `syscall.RTN_BLACKHOLE`... should I just hardcode the type here?